### PR TITLE
Improve CI

### PR DESCRIPTION
**Pull request by tomap**
* add eslint
* add dependabot for githubactions
* add lint action
* remove travis
* switch to tests for node 12, 14, 16
* update dependencies to latest minor

### CODE REVIEW

**Comment by tomap** [comment]
27 actions (matrix = 3*3*3) : 
https://github.com/tomap/hexo-fontawesome/actions/runs/1494110779
All ok

File: .github/workflows/release-drafter.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: release-drafter/release-drafter@v5
```
Comment (ertrzyiks) [inline .github/workflows/release-drafter.yml:12]: I've never used that, can't wait to see it in action

File: .github/workflows/tester.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      - name: Install Dependencies
+        run: yarn install && yarn add "hexo@5.x"
+      - name: Coverage
+        run: yarn test-cov
+        env:
+          CI: true
+      - name: Coveralls
+        uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.github_token }}
```
Comment (ertrzyiks) [inline .github/workflows/tester.yml:60]: Is the lower case version accepted? I thought that it must be upper case 
```suggestion
          github-token: ${{ secrets.GITHUB_TOKEN }}
```

**Review by ertrzyiks — COMMENTED** [review]
Sweet, the code looks good, I will test it in the evening

File: .github/workflows/release-drafter.yml (hunk 1 above)
Comment (tomap) [reply .github/workflows/release-drafter.yml:12]: Neither did I. I copied from another hexo project (cli)

File: .github/workflows/tester.yml (hunk 2 above)
Comment (tomap) [reply .github/workflows/tester.yml:60]: Seems to also work in lowercase https://coveralls.io/github/tomap/hexo-fontawesome

File: package.json [hunk 3]
```diff
@@ -4,7 +4,9 @@
   "description": "A utility function which helps to inline fontawesome SVG files.",
   "main": "index.js",
   "scripts": {
-    "test": "ava"
+    "test": "ava",
+    "test-cov": "nyc --reporter=lcovonly yarn test",
+    "eslint": "eslint"
```
Comment (ertrzyiks) [inline package.json:9]: This script it not needed, `yarn eslint` by default run the package bin. We don't have any linting rules enabled so it's a no-operation anyway.

**Comment by ertrzyiks** [comment]
Thank you! We need to make the linter check something, but it's a great starting point.

**Pull request merged**